Makefile

```diff
@@ -316,6 +316,19 @@ test-conda: air-water-vv check
 	@echo "Running air-water-vv test set 2"
 	-source ${PROTEUS_PREFIX}/bin/proteus_env.sh; MPLBACKEND=Agg py.test -n ${N} --dist=loadfile --forked -v air-water-vv/Tests/2nd_set -m ${TEST_MARKER}
 
+test-macos: air-water-vv check
```

Comment:
while this is functional, is there a way to instead place this under the usual `test` target? For example, is there a way to automatically determine that the os is mac and have it run these commands when `make test` is called?

Reply:
I suppose there is. I only added it because the pytest commands are different/and it was easier. But we can talk about this more via slack.

Reply:
You could use a conditional in the Makefile: https://www.gnu.org/software/make/manual/html_node/Conditional-Example.html#Conditional-Example